docs/cn/monitor.md

```diff
@@ -12,17 +14,17 @@ CURVE 中使用到的 bvar 数据模型有：
 
 `bvar::Adder<T>` : 计数器，默认0，varname << N相当于varname += N。
 
-`bvar::Maxer<T>` : 求最大值，默认std::numeric_limits::min()，varname << N相当于varname = max(varname, N)。
+`bvar::Maxer<T>` : 求最大值，默认std::numeric_limits::max()，varname << N相当于varname = max(varname, N)。
```

Comment:
default value is std::numeric_limits::min, because every values is bigger than it.

Reply:
> default value is std::numeric_limits::min, because every values is bigger than it.

done